frontend/src/utils/plant-naming.tsx

```diff
@@ -76,21 +78,24 @@ export function completePlantName(seed: SeedDto, plant: PlantsSummaryDto): strin
  *
  * German common names are currently not supported.
  */
-export function CompletePlantNameFormatted(props: {
+export function PlantNameFromSeedAndPlant(props: {
   seed: SeedDto;
   plant: PlantsSummaryDto;
 }): ReactElement {
-  console.assert(props.seed.plant_id === props.plant.id);
+  if (props.seed.plant_id !== props.plant.id) {
+    throw new Error('seed.plant_id must be equal to plant.id to produce a plant name');
```

Comment:
I needed an assertion here, since having a seed and a plant that are not connected does not make any sense in this context (Admittedly I should put this in the documentation).
Or to put it another way, this should never happen.

Do you know a better way of expressing an assertion in Browser-JavaScript? 
console.assert() only logs the error to the console, but does not change the program's execution.